com.github.Bleuzen.FFaudioConverter.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.Bleuzen.FFaudioConverter</id>
+  <launchable type="desktop-id">ffaudioconverter.desktop</launchable>
+  <project_license>MIT</project_license>
+  <name>FFaudioConverter</name>
+  <summary>Convert audio files</summary>
+  <description>
+    <p>Qt GUI for converting audio files with FFmpeg</p>
+  </description>
+  <categories>
+    <category>Audio</category>
+  </categories>
+  <url type="homepage">https://github.com/Bleuzen/FFaudioConverter</url>
+  <url type="bugtracker">https://github.com/Bleuzen/FFaudioConverter/issues</url>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">http://i.imgur.com/N5hhS75.png</image>
+    </screenshot>
+  </screenshots>
+</component>
```

Comment:
Add `releases`, `<content_rating type="oars-1.1"/>`, and `update_contact`

Reply:
done